packages/shared-influx-pieces/src/listeners/influxInteractionHandlerRun.ts

```diff
@@ -0,0 +1,13 @@
+import { type PieceContext, type ClientEvents } from '@skyra/http-framework';
+import { incrementInteractionCount } from 'index.js';
+import { AnalyticsListener } from 'lib/AnalyticsListener.js';
+
+export class UserAnalyticsEvent extends AnalyticsListener {
+	public constructor(context: PieceContext, options: AnalyticsListener) {
+		super(context, { ...options, event: 'interactionHandlerRun' satisfies keyof ClientEvents });
```

Comment:
Same as the other suggestions?
```suggestion
		super(context, { ...options, event: 'interactionHandlerRun' });
```